bc_obps/compliance/models/rls_configs/compliance_product.py

```diff
@@ -0,0 +1,19 @@
+from compliance.enums import ComplianceTableNames
+from rls.enums import RlsRoles, RlsOperations
+from rls.utils.helpers import generate_rls_grants
+
+
+class Rls:
+    role_grants_mapping = {
+        RlsRoles.INDUSTRY_USER: [
+            RlsOperations.SELECT,
+            RlsOperations.INSERT,
+            RlsOperations.UPDATE,
+            RlsOperations.DELETE,
+        ],
```

Comment:
I wonder if we could remove `update` and `delete` from this, since the compliance product model is created from a report_version, and immutable after that